docs/quickstarts/sso-quickstart.md

```diff
@@ -0,0 +1,158 @@
+
+## Configure SSO
+
+At this point, your add-in project has been created and contains the code that's necessary to facilitate the SSO process. Next, complete the following steps to configure SSO for your add-in.
+
+1. Go to the root folder of the project.
+
+    ```command&nbsp;line
+    cd "My Office Add-in"
+    ```
+
+2. Run the following command to configure SSO for the add-in.
+
+    ```command&nbsp;line
+    npm run configure-sso
+    ```
+
+    > [!WARNING]
+    > This command will fail if your tenant is configured to require two-factor authentication. In this scenario, you'll need to manually complete the Azure app registration and SSO configuration steps by following all the steps in the [Create a Node.js Office Add-in that uses single sign-on](../develop/create-sso-office-add-ins-nodejs.md) tutorial.
+
+3. A web browser window will open and prompt you to sign in to Azure. Sign in to Azure using your Microsoft 365 administrator credentials. These credentials will be used to register a new application in Azure and configure the settings required by SSO.
+
+    > [!NOTE]
+    > If you sign in to Azure using non-administrator credentials during this step, the `configure-sso` script won't be able to provide administrator consent for the add-in to users within your organization. SSO will therefore not be available to users of the add-in and they'll be prompted to sign-in.
+
+4. After you enter your credentials, close the browser window and return to the command prompt. As the SSO configuration process continues, you'll see status messages being written to the console. As described in the console messages, files within the add-in project that the Yeoman generator created are automatically updated with data that's required by the SSO process.
+
+## Try it out
+
+If you've created an Excel, Word, or PowerPoint add-in, complete the steps in the following section to try it out. If you've created an Outlook add-in, complete the steps in the [Outlook](#outlook) section instead.
+
+### Excel, Word, and PowerPoint
+
+Complete the following steps to try out an Excel, Word, or PowerPoint add-in.
+
+1. When the SSO configuration process completes, run the following command to build the project, start the local web server, and sideload your add-in in the previously selected Office client application.
+
+    [!INCLUDE [alert use https](../includes/alert-use-https.md)]
+
+    ```command&nbsp;line
+    npm start
+    ```
+
+2. In the Office client application that opens when you run the previous command (i.e., Excel, Word or PowerPoint), make sure that you're signed in with a user account that's a member of the same Microsoft 365 organization as the Microsoft 365 administrator account that you used to connect to Azure while configuring SSO in step 3 of the [previous section](#configure-sso). Doing so establishes the appropriate conditions for SSO to succeed.
```

Comment:
```suggestion
2. When Excel, Word, or PowerPoint opens when you run the previous command, make sure you're signed in with a user account that's a member of the same Microsoft 365 organization as the Microsoft 365 administrator account that you used to connect to Azure while configuring SSO in step 3 of the [previous section](#configure-sso). Doing so establishes the appropriate conditions for SSO to succeed.
```
Hint when to use "that" and when to omit it: We avoid "that" except when it's a relative pronoun. In other words, if can you replace "that" with "who" or "which" and it makes sense, then it stays in. This is for clarity when translating, otherwise it can usually be omitted as in the case of the second that in this step.